usefulUtilities/hiFiCalendar/entities/meetingRoomCalendar.json

```diff
@@ -0,0 +1,186 @@
+{
+  "DataVersion": 0,
+  "Entities": [
+    {
+    "id": "{990c5409-dedf-438f-b0f8-7eb0d6740b60}",
+    "type": "Text",
+    "parentID": "{c3f87945-e372-4f85-bde2-9d99d3633125}",
+    "name": "ROOM_COLOR_HEADER",
+    "position": {
+        "x": 0.0000034794211387634277,
+        "y": 0.73944091796875,
+        "z": 0.00038433074951171875
+    },
+    "dimensions": {
+        "x": 0.7599999904632568,
+        "y": 0.2322310507297516,
+        "z": 0.009999999776482582
+    },
+    "rotation": {
+        "x": -0.0000152587890625,
+        "y": -0.0000152587890625,
+        "z": -0.0000152587890625,
+        "w": 1
+    },
+    "grab": {
+        "grabbable": false
+    },
+    "collisionless": true,
+    "serverScripts": "../entityServerScripts/noFunction.js",
+    "text": "AVAILABLE",
+    "lineHeight": 0.08299999684095383,
+    "textColor": {
+        "red": 0,
+        "green": 0,
+        "blue": 0
+    },
+    "backgroundColor": {
+        "red": 125,
+        "green": 255,
+        "blue": 125
+    },
+    "leftMargin": 0.029999999329447746,
+    "topMargin": 0.14000000059604645
+},
+    {
+    "id": "{96e394bf-947d-4263-8538-9d5a38148378}",
+    "type": "Text",
+    "parentID": "{c3f87945-e372-4f85-bde2-9d99d3633125}",
+    "name": "ROOM_CLOCK",
+    "userData": "{\"grabbableKey\":{\"grabbable\":false}}",
+    "position": {
+        "x": -0.08176710456609726,
+        "y": 0.7408447265625,
+        "z": 0.018096923828125
+    },
+    "dimensions": {
+        "x": 0.5278285145759583,
+        "y": 0.17838022112846375,
+        "z": 0.009999999776482582
+    },
+    "rotation": {
+        "x": 0.0001678466796875,
+        "y": -0.0000152587890625,
+        "z": 0.0000762939453125,
+        "w": 1
+    },
+    "grab": {
+        "grabbable": false
+    },
+    "collisionless": true,
+    "serverScripts": "http://hifi-content.s3-us-west-1.amazonaws.com/rebecca/LoadTest/clockPST.js",
```

Comment:
what's this?

Reply:
This is the clock at the top of the schedules... I didn't realize this wasn't released code... I will investigate.